js/deer-render.js

```diff
@@ -104,14 +104,15 @@ RENDER.element = function (elem, obj) {
  * @param {Object} obj some json to be drawn as JSON
  * @param {Object} options additional properties to draw with the JSON
  */
+
 DEER.TEMPLATES.json = function (obj, options = {}) {
     let indent = options.indent || 4
     let replacer = (k, v) => {
         if (DEER.SUPPRESS.indexOf(k) !== -1) return
         return v
     }
     try {
-        return `<pre>${JSON.stringify(obj, replacer, indent)}</pre>`
+        return `<pre style="border: 2px solid red;">${JSON.stringify(obj, replacer, indent)}</pre>`
```

Comment:
This is not an improvement.  Please look into CSS styling for JSON previews.  Make sure a button is available to copy the JSON to the clipboard or download it to the local machine.